src/components/templates/ConversationSpace/ConversationSpace.tsx

```diff
@@ -135,23 +134,18 @@ export const ConversationSpace: React.FC<ConversationSpaceProps> = ({
               )}
             </div>
           </div>
-          <div className="seated-area">
-            {userWithId && (
-              <TablesUserList
-                setSeatedAtTable={setSeatedAtTable}
-                seatedAtTable={seatedAtTable}
-                venueId={venue.id}
-                TableComponent={TableComponent}
-                joinMessage={venue.hideVideo === false}
-                customTables={tables}
-                defaultTables={CONVERSATION_TABLES}
-                showOnlyAvailableTables={showOnlyAvailableTables}
-                venue={venue}
-                template={VenueTemplate.conversationspace}
-                user={userWithId}
-              />
-            )}
-          </div>
+          {userWithId && (
+            <TableGrid
+              venueId={venue.id}
+              joinMessage={venue.hideVideo === false}
```

Comment:
Quite a specific check.
Would there be other values like `undefined` and/or `null` in play here?
If so, it might use a comment with the reasoning behind it.

```suggestion
              joinMessage={venue.hideVideo === false}
```

Reply:
This was the existing check. I expect that when we get to redoing the conversation space this will be destroyed.